src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -16,6 +16,7 @@ class AuthApplicationSection extends React.Component {
             <p key="claims"><a href="/track-claims">Check your claim and appeal status</a></p>
             <p key="post-911"><a href="/education/gi-bill/post-9-11/ch-33-benefit">Get your Post-9/11 GI Bill statement of benefits</a></p>
             <p key="vic"><a href="/veteran-id-card">Apply for a Veteran ID Card</a></p>
+            <p key="letters"><a href="/download-va-letters">Download VA Letters</a></p>
```

Comment:
Same here as above, I think.

Reply:
The link URL you mean? It should go to the intro page (`/download-va-letters`) not the app (`/download-va-letters/letters`)

Reply:
Thanks for the helpful context, @egarreau.